sdk/boms/azure-sdk-bom/CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Release History
 
+## 1.2.38 (2025-09-03)
+
+### Dependency Updates
+
+- Updated Azure SDK dependency versions to the latest releases.
```

Comment:
Looks like we added Health Deidentification and Event Grid SystemEvents packages to the BOM. It should be mentioned in the changelog.

Reply:
@srnagar I'll add Event Grid SystemEvents. Health Deidentification is already in the changelog (2025-04-08)